scripts/voqutil

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+import sys
+
+from natsort import natsorted
+from tabulate import tabulate
+from utilities_common import multi_asic as multi_asic_util
+from swsscommon import swsscommon
+
+# mock the redis for unit test purposes #
+try:
+    if os.environ["UTILITIES_UNIT_TESTING"] == "1":
+        modules_path = os.path.join(os.path.dirname(__file__), "..")
+        tests_path = os.path.join(modules_path, "tests")
+        sys.path.insert(0, modules_path)
+        sys.path.insert(0, tests_path)
+        import mock_tables.dbconnector
+        if os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] == "multi_asic":
+            import mock_tables.mock_multi_asic
+            mock_tables.dbconnector.load_namespace_config()
+        else:
+            mock_tables.dbconnector.load_database_config()
+
+except KeyError:
+    pass
+
+
+# ========================== Common VOQ util logic ==========================
+
+SYSTEM_PORT_TABLE_PREFIX = "SYSTEM_PORT_TABLE:"
+SYSTEM_PORT_ID = "system_port_id"
+SYSTEM_PORT_CORE = "core_index"
+SYSTEM_PORT_CORE_PORT = "core_port_index"
+SYSTEM_PORT_SPEED = "speed"
+SYSTEM_PORT_SWITCH_ID = "switch_id"
+
+SYSTEM_NEIGH_TABLE_PREFIX = "SYSTEM_NEIGH|"
+SYSTEM_NEIGH_MAC = "neigh"
+SYSTEM_NEIGH_ENCAP_INDEX = "encap_index"
+
+SYSTEM_LAG_TABLE_PREFIX = "SYSTEM_LAG_TABLE|"
+SYSTEM_LAG_MEMBER_TABLE_PREFIX = "SYSTEM_LAG_MEMBER_TABLE|"
+SYSTEM_LAG_ID = "lag_id"
+SYSTEM_LAG_SWITCH_ID = "switch_id"
+
+def parse_systemintf_in_filter(intf_filter):
+    intf_fs = []
+
+    if intf_filter is None:
+        return intf_fs
+
+    fs = intf_filter.split(',')
+    for x in fs:
+        intf_fs.append(x.strip())
+
+    return intf_fs
+
+def appl_db_keys_get_system_port(appl_db, system_port_name):
+    """
+    Get APPL_DB SYSTEM_PORT_TABLE Keys
+    """
+    if system_port_name is None or system_port_name == "":
+        appl_db_keys = appl_db.keys(appl_db.APPL_DB, "SYSTEM_PORT_TABLE:*")
+    else:
+        appl_db_keys = appl_db.keys(appl_db.APPL_DB, "SYSTEM_PORT_TABLE:%s" % system_port_name)
+
+    return appl_db_keys
+
+def appl_db_system_port_status_get(appl_db, system_port_name, status_type):
+    """
+    Get the system port status
+    """
+    full_table_id = SYSTEM_PORT_TABLE_PREFIX + system_port_name
+    status = appl_db.get(appl_db.APPL_DB, full_table_id, status_type)
+    if status is None:
+        return "N/A"
+    if status_type == SYSTEM_PORT_SPEED and status != "N/A":
+       status = '{}G'.format(status[:-3])
+    return status
+
+def chassis_app_db_keys_get_system_neigh(chassis_app_db):
+    """
+    Get CHASSIS_APP_DB SYSTEM_NEIGH table Keys
+    """
+    system_neigh_keys = chassis_app_db.keys(chassis_app_db.CHASSIS_APP_DB, "SYSTEM_NEIGH|*")
```

Comment:
Use `SYSTEM_NEIGH_TABLE_PREFIX ` iso `SYSTEM_NEIGH|` here ?

Reply:
Fixed, here and in other places also.